frame/support/src/traits/tokens/nonfungible_v2.rs

```diff
@@ -42,15 +45,23 @@ pub trait Inspect<AccountId> {
 	/// Returns the attribute value of `item` corresponding to `key`.
 	///
 	/// By default this is `None`; no attributes are defined.
-	fn attribute(_item: &Self::ItemId, _key: &[u8]) -> Option<Vec<u8>> {
+	fn attribute(
+		_item: &Self::ItemId,
+		_key: &[u8],
+		_namespace: &AttributeNamespace<AccountId>,
+	) -> Option<Vec<u8>> {
```

Comment:
should arg order logically go item, namespace, key? as there's many keys in a namespace

Reply:
good point, changing that

Reply:
done